libafl/src/events/launcher.rs

```diff
@@ -808,19 +815,11 @@ where
                 broker.inner_mut().connect_b2b(remote_broker_addr)?;
             };
 
-            let exit_cleanly_after = NonZeroUsize::try_from(self.cores.ids.len()).unwrap();
-
-            broker
-                .inner_mut()
-                .set_exit_cleanly_after(exit_cleanly_after);
+            broker.set_exit_after(exit_cleanly_after);
 
             brokers.add(Box::new(broker));
         }
-
-        log::debug!(
-            "Brokers have been initialized on port {}.",
-            std::process::id()
-        );
+        log::debug!("Brokers have been initialized; pid {}.", std::process::id());
```

Comment:
a) is the pid helpful
b) the pid is only for one broker, not for "Broker_s_"

Reply:
yes pid is helpful for debugging